src/components/CheckBox/CheckboxList.tsx

```
@@ -0,0 +1,312 @@
+    return (
+      <CheckBox
+        ref={ref}
+        id={inputId}
+        name={name}
+        value={value}
+        checked={isChecked}
+        disabled={isDisabled}
+        size={size}
+        state={currentState}
+        className={className}
+        onChange={(e) => {
```

ESLint

[error] 'e' is defined but never used. Allowed unused args must match /^_/u.

(@typescript-eslint/no-unused-vars)